drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -859,6 +859,26 @@ void AudioDriverWASAPI::thread_func(void *p_udata) {
 								r = read_sample(ad->audio_input.format_tag, ad->audio_input.bits_per_sample, data, j * 2 + 1);
 							} else if (ad->audio_input.channels == 1) {
 								l = r = read_sample(ad->audio_input.format_tag, ad->audio_input.bits_per_sample, data, j);
+							} else if (ad->audio_input.channels >= 2) {
+								int channels = ad->audio_input.channels;
+								for (int ch = 0; ch < channels - 1; ch++) {
+									int32_t sample = read_sample(ad->audio_input.format_tag, ad->audio_input.bits_per_sample, data, j * channels + ch);
+									if (ch % 2 == 0) {
+										r += sample;
+									} else {
+										l += sample;
+									}
```

Comment:
From what I know, the assumption is naive. Like, depending on the number of channels, sometimes it's left, right, or even center.